### ISSUE TITLE
minify builds

### ISSUE THREAD

**Issue by maxlath** [cozy/cozy-proxy#203]
the interface when registering feels a bit slow, maybe because lots of things are happening in the background at the same time, but some optimization in the builds could come for (almost) free and might help to make it feel more snappy:
`uglifyjs --compress --mangle -- vendor.js > vendor.min.js`

```
772K	vendor.js
280K	vendor.min.js
```

the only downside being that debugging gets sensibly more difficult with those 10000+ characters per line

**Comment by clochix** [comment]
Hi @maxlath,
Thanks the this great idea, see also #201.

@aenario, @m4dz any objection against minifying le client js resources ?

**Comment by m4dz** [comment]
I totally agree. To be honest, it seems to be a mistake in our Brunch production env config, because I was certain that we already minify it. I'll check that right now. Thanks!

**Comment by frankrousseau** [comment]
I think we met problems with minifying and version control. I don't remember why we desactivated this feature.

**Comment by aenario** [comment]
The problem was to make debugging impossible in production because there was a bug in brunch vs uglify on pipelining sourcemaps (coffee -> js -> µjs). 

I am not sure how well it will interact with @clochix works on piping errors from client to servers also.

**Comment by clochix** [comment]
Yes, minifying will have an impact on reading client side errors logs, but I think it is worth doing it, because it will harden a bit our work, but make our users life smoother.
